### PR TITLE
Added split code blocks to home page

### LINKED ISSUE #350: Code blocks on homepage
The four code blocks on the homepage should be two in one row, instead of wasting a bunch of space:

screen shot 2016-11-17 at 18 03 09

Those code blocks could easily fit next to each other instead of having a massive amount of horizontal space, like this:

| Write your code | |
| ------------- | ------------- |
|  import bar from 'foo';       | export function bar()           |
| **Bundle with webpack** | |
| module.exports = {}      | &lt;html&gt; |

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves #350

Now you can write HTML directly into the markdown files.

### CODE REVIEW

**Comment by bebraw** [comment]
Looks good to me. Do you want to add anything else to the PR?

**Comment by bebraw** [comment]
Can you rebase against `master`? The development happens against it now (faster deployments).

**Comment by NejcZdovc** [comment]
@bebraw rebased

File: utilities/markdown.js [hunk 1]
```diff
@@ -112,23 +113,63 @@ module.exports = function(section) {
...
+    // Split code in markdown, so that HTML inside code is not parsed
+    var codeArray = t.text.split(/(```(.|\n)*```)/g).filter(v => (v !== '' && v !== '\n'));
+
+    // if only one item in codeArray, then it's already parsed
+    if(codeArray.length == 1) {
+      return t;
+    }
+
+    codeArray.forEach(item => {
+      // if item is not code, then check for html tags and parse accordingly
```
Comment (SpaceK33z) [inline utilities/markdown.js]: What about code in backticks (e.g. \`<script>\`)?

Comment (NejcZdovc) [inline utilities/markdown.js]: I still need to cover this scenario, when we use only single backtick

Comment (NejcZdovc) [inline utilities/markdown.js]: Single backtick is now covered as well.

**Comment by NejcZdovc** [comment]
@SpaceK33z can you look at it again please

**Pull request merged**